src/layout/src/components/App/AppHeaderWrapper.js

```diff
@@ -257,10 +201,31 @@ class AppHeaderWrapper extends Component<Props, State> {
     window.removeEventListener('resize', this.handleOnResize.bind(this));
```

Comment:
Is binding `this` necessary?

Reply:
Yes, because we want `this` to refer to `AppHeaderWrapper` inside `handleOnResize` so that we can access the refs. Otherwise, `this` refers to the window.